src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -187,6 +188,17 @@ default String ignoredNames() {
         return "";
     }
 
+    @ConfigItem(
```

Comment:
Move the code to under the `ignoredNames` setting so the settings are also grouped in this file.

Reply:
I don't know what you mean? The new config is at the lines 191-200 directly below `ignoredNames` at line 180-189?

Reply:
You're right, my bad!